.changeset/great-kangaroos-confess.md

```diff
@@ -0,0 +1,19 @@
+---
+'xstate': major
+---
+
+You can now add a `systemId` to spawned actors to reference them anywhere in the system.
+
+```ts
+const machine = createMachine({
+  // ...
+  context: ({ spawn }) => ({
+    actorRef: spawn(
+      createMachine({
+        // ...
+      }),
+      { systemId: 'actorRef' }
+    )
+  })
+});
+```
```

Comment:
Showing how to reference the actor by `systemId` would add a lot of value here

Reply:
Will do this in the docs